packages/lib/src/components/groups/lume-bar-group/lume-bar-group.vue

```diff
@@ -122,7 +122,7 @@ async function handleInternalHover(groupIndex: number) {
   emit('lume__internal--hover', groupIndex);
 
   if (__VUE_VERSION__ === 2) {
-    await busEmit('lume__internal--hover', groupIndex);
+    emitInternalEvent('lume__internal--hover', groupIndex);
```

Comment:
Shouldn't all `group`s emit this event?

Reply:
haha I see!
Capturing events in line group is not as straight forward as it's in bar groups 😅 

For bar-groups, the `${chartID}-lume__internal--hover` mouseover event can be captured over every mouseenter of a group containing lume-bar(s).

Every group in bar group actually represents which group we're hovering on.

But in the case of lume-line-group - it's a group of lines and points.

So even though there are 20 points, if the group has 3 lines, then number of groups are still 3.
If we decide to add mouseover event for the points, we still can't cover the scenario of hovering on position between the axis and point. So the event will not be trigerred if the consumer doesn't hover on the point.